src/postmate.js

```diff
@@ -3,20 +3,20 @@
  * The type of messages our frames our sending
  * @type {String}
  */
-const MESSAGE_TYPE = 'application/x-postmate-v1+json'
+export const message_type = 'application/x-postmate-v1+json'
 
 /**
  * hasOwnProperty()
  * @type {Function}
  * @return {Boolean}
  */
-const hasOwnProperty = Object.prototype.hasOwnProperty
+export const hasOwnProperty = Object.prototype.hasOwnProperty
```

Comment:
I don't think we need to export this since we would technically just be testing `Object.prototype.hasOwnProperty` which I don't recommend